dipy/denoise/denspeed.pyx

```diff
@@ -103,10 +103,9 @@ def _nlmeans_3d(double [:, :, ::1] arr, double [:, :, ::1] mask,
 
     # move the block
     with nogil, parallel():
-
-        for i in prange(B, I - B):
-            for j in range(B, J - B):
-                for k in range(B, K - B):
+        for i in range(B+P, I - B-P):
```

Comment:
space between operators
All pep8 should be supported in Cython files except from the line width limitation.